src/components/Section/ProposalResultSection.tsx

```diff
@@ -37,15 +36,14 @@ export default function ProposalResultSection({
   disabled,
   votes,
   changingVote,
-  votingPower,
   onChangeVote,
   onVote,
   onOpenVotesList,
   ...props
 }: ProposalResultSectionProps) {
   const t = useFormatMessage()
-  const choices = proposal?.snapshot_proposal?.choices || []
-  const results = useMemo(() => calculateResult(choices, votes || {}), [proposal, choices, votes])
+  const choices = useMemo(() => proposal?.snapshot_proposal?.choices || [], [proposal?.snapshot_proposal?.choices])
```

Comment:
I don't see this useMemo as necessary. If the [] worries you, you can move it outside the function as a constant
`const choices = proposal?.snapshot_proposal?.choices || EMPTY_CHOICES`

Reply:
I included the useMemo because of this warning and I can't move the constate out of the function because 'proposal' is an argument of it.

![image](https://user-images.githubusercontent.com/45410089/169307700-211bb2d7-b5c2-4011-823f-732fff0eb3d1.png)

Reply:
the problem was the empty array as you mentioned!